l10n_eu_nace/wizard/res_partner_industry_eu_nace_wizard_view.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2022 Moduon
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html). -->
+<odoo>
+
+    <record id="partner_industry_nace_wizard_view_form" model="ir.ui.view">
+        <field name="name">re.partner.industry.nace.wizard.view.form</field>
+        <field name="model">res.partner.industry.eu.nace.wizard</field>
+        <field name="arch" type="xml">
+            <form string="Import NACE industries">
+                <sheet>
+                    <group>
+                        <button
```

Comment:
Maybe the button should be in the footer. Also, in the sheet it would be nice to put an informative text.

Reply:
Done!